include/osquery/tables.h

```diff
@@ -459,6 +464,9 @@ using ConstraintSet = std::vector<std::pair<std::string, struct Constraint>>;
 /// Keep track of which columns are used
 using UsedColumns = std::unordered_set<std::string>;
 
+/// Keep track of which columns are used, as a bitset
+using UsedColumnsBitset = std::bitset<64>;
```

Comment:
May I ask you to define the max number of columns somewhere here as constexpr value?

Reply:
Yep, will do.